Add GPU support for Linux Azure Container Instances (#3053)
terraform-providers/terraform-provider-azurerm: website/docs/r/container_group.html.markdown

* `memory` - (Required) The required memory of the containers in GB. Changing this forces a new resource to be created.

> ~> **Note:** Gpu resources are currently only supported in Linux containers.

Reviewer: we tend to put `note` fields below the field they're talking about - could we move this below the line below?

Author: Done, thanks.

> * `gpu` - (Optional) Specify to deploy the container with a GPU resource.

Reviewer: for consistency with the other resources could we make this:

```suggestion
* `gpu` - (Optional) A `gpu` block as defined below.
```

Author: Done, thanks.

* `protocol` - (Required) The network protocol associated with port. Possible values are `TCP` & `UDP`.

--

A `gpu` block supports:

> * `gpu_count` - (Required) The number of GPUs. Allowed values are 1, 2, or 4.

Author: Done, thanks.

> * `gpu_sku` - (Required) The GPU SKU. Allowed values are K80, P100, or V100.

Author: Done, thanks.
